src/sentry/api/endpoints/auth_index.py

```diff
@@ -33,10 +32,6 @@
 
 PREFILLED_SU_MODAL_KEY = "prefilled_su_modal"
 
-DISABLE_SU_FORM_U2F_CHECK_FOR_LOCAL = getattr(
-    settings, "DISABLE_SU_FORM_U2F_CHECK_FOR_LOCAL", False
-)
```

Comment:
Forgot to remove this in an earlier PR b/c we no longer use this in this file

Reply:
do we need to remove it from https://github.com/getsentry/sentry/pull/66364/files#diff-dcd4801baa22c78923c11bf52c23caa202d80d2990d78b0500507f78d9950c95R71 as well?

Reply:
no I changed it recently so we use it in that file, which is what i'm debugging now